rfswarm_reporter/rfswarm_reporter.py

```diff
@@ -3943,9 +3943,13 @@ def xhtml_sections_table(self, elmt, id):
 					# for val in vals:
 					for col in cols:
 						if col not in ["Colour"]:
-							val = str(row[col]).strip()
 							td = etree.SubElement(tr, 'td')
-							td.text = str(val)
+							val = str(row[col]).strip()
+							# Exclude illegal characters out of xml
+							illegal_xml_chars_re = re.compile(u'[\x00-\x08\x0b\x0c\x0e-\x1F\uD800-\uDFFF\uFFFE\uFFFF]', re.UNICODE)
```

Comment:
Can you move the variable `illegal_xml_chars_re` as a class variable in the top of ReporterCore, so it only needs to be statically defined once, then all of these
```
val = illegal_xml_chars_re.sub('', val)
```
become
```
val = self.illegal_xml_chars_re.sub('', val)
```